lib/index.ts

```diff
@@ -7,6 +7,28 @@ export * from './Zoom/'
 // Types
 export * from './BaseProvider'
 
+/**
+ * Instancia de log de una petición realizada.
+ */
+type Log = {
+  /**
+   * Cabeceras de la petición.
+   */
+  headers?: any
```

Comment:
```suggestion
  headers?: {
    [key: string]: string | number | boolean | Date
  }
```
Así podría no quedar con any y headers es un objeto con `[key: value]`

Reply:
@SeGonzalezR Toda la razón 👏 , cambio aplicado en la ultima actualización